src/main/java/ssl/ois/timelog/service/role/get/GetRoleUseCase.java

```diff
@@ -0,0 +1,27 @@
+package ssl.ois.timelog.service.role.get;
+import ssl.ois.timelog.service.exception.role.GetRoleErrorException;
+import ssl.ois.timelog.service.repository.user.UnitRepository;
+
+import org.springframework.stereotype.Service;
+
+import ssl.ois.timelog.model.team.Role;
+import ssl.ois.timelog.service.exception.DatabaseErrorException;
+
+@Service
+public class GetRoleUseCase {
```

Comment:
get role from team entity instead of repo

Reply:
if we want to get the role from the team entity
we have to new a team entity
idk what is the difference between
get role from repo and
first get team entity from repo and then get role from team

I know it's because of the domain, but what will occur if we get it directly from repo?